vendor/github.com/kubernetes/repo-infra/kazel/BUILD

```diff
@@ -1,18 +1,4 @@
-package(default_visibility = ["//visibility:public"])
-
-licenses(["notice"])
-
-load(
-    "@io_bazel_rules_go//go:def.bzl",
-    "go_binary",
-    "go_library",
-)
-
-go_binary(
-    name = "kazel",
-    embed = [":go_default_library"],
-    tags = ["automanaged"],
-)
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
/cc @ixdy 
This doesn't look right.

Reply:
Do you mean the formatting looks incorrect? Or something else?

Reply:
@vantuvt The fact that this file is being changed at all. It might have to do with a change that @ixdy made recently, so that's why I've cc'd him.

Reply:
@cblecker I see. Thanks for clarifying.

Reply:
yeah, https://github.com/kubernetes/kubernetes/pull/62499 will fix this, but I'm still waiting on `pkg/` approval.